ledger-core/virtual/integration/delegated_finished_test.go

```diff
@@ -239,19 +240,11 @@ func TestVirtual_SendDelegatedFinished_IfPulseChanged_Without_SideEffect(t *test
 		server.SendPayload(ctx, &pl)
 	}
 
-	{
-		select {
-		case <-delegateDone:
-		case <-time.After(10 * time.Second):
-			require.FailNow(t, "timeout")
-		}
-
-		select {
-		case <-server.Journal.WaitAllAsyncCallsDone():
-		case <-time.After(10 * time.Second):
-			require.FailNow(t, "timeout")
-		}
-	}
+	testutils.WaitSignalsTimed(
```

Comment:
Besides, I'd prefer common helper that returns `true`/`false`, use it in tests like
```
require.True(t, testutils.WaitSignalsTiimed(), "timeout")
```

Reply:
This is annoying boilerplate that my goal was to get rid off. I was even considering hiding duration in testutils package to avoid repeating 10*time.Seconds. May be name should be `testutils.RequireSignalsTimed`. What do you think?

fixed problem of locating point of error by using `t.Helper()` method. Now "timeout" error trace will show point in test, not in the utility function.

Reply:
Sure.